installer/scripts/service_control

```diff
@@ -283,12 +495,16 @@ enable_omsagent_service()
             elif [ -x /sbin/chkconfig ]; then
                 /sbin/chkconfig --add $OMSAGENT_WS > /dev/null
             else
-                echo "Could not find a service controller to configure the OMS Agent Service."
-                exit 1
+                echo "FATAL:  Could not find a service controller to configure the OMS Agent Service."
+                exit 9
             fi
         fi
-        touch  $CONF_DIR/.service_registered
+        WS_STATUS=$?
+        if [ "$WS_STATUS" -eq 0 ]; then
+            touch  $CONF_DIR/.service_registered
```

Comment:
nit: remove extra space in this line

Reply:
Got it.  That was where I lost my build machine.  ;^)